lib/matplotlib/tests/test_table.py

```diff
@@ -270,3 +270,24 @@ def test_table_dataframe(pd):
     for r, (index, row) in enumerate(df.iterrows()):
         for c, col in enumerate(df.columns if r == 0 else row.values):
             assert table[r if r == 0 else r+1, c].get_text().get_text() == str(col)
+
+
+# Test function for fontsize in table
```

Comment:
I would move this comment inside the test and make it more explicit e.g. "Test that the passed fontsize propagates to cells".  I also think the comments currently in the test are unnecessary as they are just stating what the code does, which someone familiar with Matplotlib can tell by reading the code itself.